### ISSUE TITLE
oca-clabot : False positive messages

### ISSUE THREAD

**Issue by leemannd** [OCA/maintainer-tools#435]
### Current behavior

1. User `lambda_user` Signs the CLA, open a PR and git it merged into the OCA repositories
2. `lambda_user` Leaves his company and get his github account deleted (or he just deletes his account because he wants to
3. Someone else migrates the module containing the commit.

You will endup with the `oca-clabot` raising an CLA warning.
Here you find an example: https://github.com/OCA/sale-workflow/pull/983#issuecomment-549374969

### Expected behavior

The CLA warning should not be prompted.
Sidenote: I'm not 100% sure this is an issue for you, nor if there is a clean way to manage it.

What do you think?

**Comment by pedrobaeza** [comment]
I agree we should keep the CLA in that cases, and detect by mail, not by GitHub user, but currently the technical limitation is that one, that it's linked to the GitHub user, so if the user unlinks the corporate mail address from his/her user, we get the error.

**Comment by leemannd** [comment]
Ok. Thanks for your answer. I taught there could be technical issues but I wasn't sure.

Feel free to close the issue if you see no solution or to keep it open as reminder if you want to do something with it.

**Comment by pedrobaeza** [comment]
@gurneyalex @sbidoul can complement my answer to see if we can find a way out.
